### ISSUE TITLE
Introduced instability/panic on Kernel 6.5

### ISSUE THREAD

**Issue by a17t** [strongtz/i915-sriov-dkms#139]
After fixing some minor compile bugs (see previous PRs) this module built for kernel 6.5.11 on Proxmox 8.1.
The idea being, to pass through the GPU of the intel n100 to VMs, enabling hardware encoding.

Unfortunately with the module loaded and enabled, the system panics every so often (4-12h). I tried getting a dmesg or any further log output, which was impossible up to now. (System runs headless) 

I will enrich this issue with the panic (I will connect a screen and try to get that panic on screen). In case it  turns out, it is not this driver, I will of course close this. 

Anyone else experiencing similar behaviour.

Hardware used:
Intel N100 (13 Gen)
32GB DDR5 Ram

(Random Aliexpress board...)

**Comment by mayanez** [comment]
Can confirm I have the same issue. 

I tried various methods to try and get logging capturing any issue, but there was never any I could identify. It occurs as a hard shutdown from what I can tell. I'm running a i5-13400k. I've since disabled the DKMS and stability has been rock solid, so I can rule out bad HW.

**Comment by a17t** [comment]
Unfortunately did not have the time to investigate further. But if disabling dkms solved the instability for you I will give it a try. I anyway want to run that all on a headless machine.

**Comment by BenSchweikert** [comment]
Same here with a ASRock Z690 Phantom Gaming 4 and Intel Core i5-12500. I checked PowerSaving Options, Bios settings and Memtest. I am using now iGPU passthrough to a VM which is solid. So I currently think it is related to the dkms. I checked several Kernel Versions starting with 6.2 up to 6.5 with a Proxmox Host. The hard reboots occured with all kernels but I think the frequency was different.
